README.md

```diff
@@ -4,7 +4,12 @@ This is a seed project to get us started with prototyping.
 
 ## Installation
 
-First install the toolchain with [NPM](https://www.npmjs.org/):
+First install the command line tools globally with [NPM](https://www.npmjs.org/):
+```shell
+npm install --global gulp uglify-js2
```

Comment:
Okay, yeah the initial `gulp` error makes sense because if you don't install it globally you won't be able to run it as `gulp build`. Rather you'll need to have NPM run it for you (e.g. `npm run gulp build`).

I'll take a closer look at your above error when I get home in a couple of hours. Hopefully it's just a matter of adding a new devDependency to the package.json and mentioning the note about `npm run` in the README. :)